lib/bindConnectors/bindMessageHooks.js

```diff
@@ -38,23 +48,33 @@ module.exports = function (config, options) {
 			}
 
 			if (_.isFunction(message.request) || _.isObject(message.request)) {
-				messageHooks[message.name+'_request'] = bindTriggerRequest(message, options);
+				var requestOpName = message.name + '_request';
+				messageHooks[requestOpName] = bindTriggerRequest(message, options);
+				if (process.env.NODE_ENV === 'test') {
```

Comment:
Rather than checking the environment variables here, could you use the options object and check for `options.test` and then check for the environment variable in the root `index.js` https://github.com/trayio/falafel/blob/20c7edf9a3b3117ce89a25f4ee2e6b98a2401b0a/lib/index.js#L73-L76 

This means that we could have the option of passing in the options without having to change the environment variables.

Reply:
good suggestion